src/platform-includes/crons/setup/php.mdx

```diff
@@ -60,9 +63,64 @@ If your job execution fails, you can:
 // 🔴 Notify Sentry your job has failed:
 $event = Event::createCheckIn();
 $event->setCheckIn(new CheckIn(
-    id: $checkIn->getId(),
     monitorSlug: '<monitor-slug>',
     status: CheckInStatus::error(),
+    id: $checkIn->getId(),
+    release: '1.0.0', // Optional release of your application
+    environment: 'production', // Optional environment your cron is running on
+    duration: 10, // Optional duration in seconds
+));
+SentrySdk::getCurrentHub()->captureEvent($event);
+```
+
+## Upserting Cron Monitors
+
+Instead of [creating and configuring](https://sentry.io/crons/create/) your Monitor upfront,
+you can programmatically create and update your Monitors directly in code.
```

Comment:
```suggestion
You can create and update your Monitors programmatically with code
rather than [creating and configuring them in Sentry.io](https://sentry.io/crons/create/).
```

Reply:
The link originally felt jarring here, I wouldn't have expected a link to the product. I think this is more intuitive!